src/nora/components/NoraDrawer/NoraDrawer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Drawer } from '../../../components/index'
+import styles from './NoraDrawer.module.scss'
+
+/**
+ * Drawer component used to progressively disclose information when toggled
+ *
+ * @public (or @private?)
+ *
+ * @param {ReactNode} children - Children to render within the drawer
+ * @param {boolean} isOpen - indicates if drawer is open or not
+ * @param {string} labelCopy - Copy to display in the drawer's header
+ *
+ * @return {JsxElement}
+ */
+export const NoraDrawer = ({
+  children,
+  isOpen,
+  onDismiss,
+  position,
+  labelCopy,
+  closeCopy,
+}) => {
+  return (
+    <Drawer
+      className={styles.NoraDrawer}
+      onDismiss={onDismiss}
+      isOpen={isOpen}
+      position={position}
+    >
+      <header className={styles.Header}>
+        <button onClick={() => onDismiss(false)} className={styles.Close}>
+          {closeCopy}
```

Comment:
Not sure if it's the styling but it seems like the focus on the button or the text might not be centered properly?
![image](https://user-images.githubusercontent.com/12839832/75494820-eb2d1c00-5971-11ea-8e95-0eaf6ea71711.png)

Reply:
![image](https://user-images.githubusercontent.com/142403/75497092-ff741780-5977-11ea-93b6-e3a3524f0406.png)

in latest

Reply:
woot woot!